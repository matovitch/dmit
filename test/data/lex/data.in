'plop'test'plop'
'test'test'test'
'test'test''test'
'test'test'''test'
'test'test''''test'
''test''
"test"
''''