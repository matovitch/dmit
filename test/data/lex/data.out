{"tokens" :["START_OF_INPUT","DATA","WHITESPACE","DATA","WHITESPACE","DATA","WHITESPACE","DATA","WHITESPACE","DATA","WHITESPACE","DATA","WHITESPACE","DATA","WHITESPACE","DATA","END_OF_INPUT"],"offsets":[111,96,94,79,77,61,59,42,40,22,20,13,11,5,4,1,0]}